spec/policy_machine_storage_adapters/active_record_spec.rb

```diff
@@ -246,6 +246,12 @@
       end
 
       describe 'accessible_objects' do
+        before do
+          allow(PolicyMachineStorageAdapter).to receive(:postgres?).and_return(false)
+          expect_any_instance_of(PolicyMachineStorageAdapter::ActiveRecord)
```

Comment:
what's this for? `expect`s shouldn't really be in a `before` block. it looks like this one is a real expectation so should go in an example (`it`) block.

Reply:
(same goes for similar `expect_any_instance_of...not_to receive` instance below)